plugins/src/main/java/org/smartregister/fhir/gateway/plugins/Constants.java

```diff
@@ -20,19 +20,19 @@ public class Constants {
     public static final String PROXY_TO_ENV = "PROXY_TO";
     public static final String SYNC_STRATEGY = "syncStrategy";
     public static final String TAG_SEARCH_PARAM = "_tag";
-
     public static final String AUTHORIZATION = "Authorization";
-
     public static final String KEYCLOAK_UUID = "keycloak-uuid";
     public static final String IDENTIFIER = "identifier";
-
     public static final String PAGINATION_PAGE_SIZE = "_count";
-
     public static final String PAGINATION_PAGE_NUMBER = "_page";
-
     public static final int PAGINATION_DEFAULT_PAGE_SIZE = 20;
-
     public static final int PAGINATION_DEFAULT_PAGE_NUMBER = 1;
+    public static final String SYNC_LOCATIONS = "_syncLocations";
+    public static final String RELATED_ENTITY_LOCATION = "RelatedEntityLocation";
+    public static final String RELATED_ENTITY_TAG_URL_ENV = "RELATED_ENTITY_TAG_URL";
+    public static final String DEFAULT_RELATED_ENTITY_TAG_URL =
+            "https://smartregister.org/related-entity-location-tag-id";
+    public static final String ROLE_ALL_LOCATIONS = "ROLE_ALL_LOCATIONS";
```

Comment:
This is the name of the role as described in the issue. I don't think it is standard for keycloak roles to be preceded by `ROLE`. Should it he kept as `ROLE_ALL_LOCATIONS` or `ALL_LOCATIONS`?

Reply:
Good question. I'm okay with dropping it or if we need a prefix I'd go with `OPENSRP_ALL_LOCATIONS`. But I'm thinking if we do that then we need to prefix all our roles for consistency. Dropping it might be best for now cc @pld @dubdabasoduba